app/src/main/java/no/ntnu/klubbhuset/service/UserService.java

```diff
@@ -36,23 +49,51 @@ public JSONObject createNewUser(String firstname, String lastname, String email,
         } catch (JSONException e) {
             e.printStackTrace();
         }
+        String requestBody = newUserJson.toString();
 
-
-        JsonObjectRequest jsonObjectRequest = new JsonObjectRequest(Request.Method.POST, URL, newUserJson,
+        StringRequest jsonObjectRequest = new StringRequest(Request.Method.POST, URL,
                 response -> {
                     Log.d(TAG, "createNewUser: got response" + response);
-                    // todo. Start main activity. send response to the activity
+                    Toast.makeText(context,"Profile created. You can now login", Toast.LENGTH_LONG).show();
+                    Intent intent = new Intent(context, LoginActivity.class);
+                    context.startActivity(intent);
                 },
                 error -> {
                     Log.d(TAG, "createNewUser: error");
                     if ( error != null) {
                         // todo handle error
-                        Log.d(TAG, "createNewUser: Error" + error);
+                        Log.d(TAG, "createNewUser: Error: " + error);
                     }
-                });
+                }) {
+            @Override
+            public String getBodyContentType() {
+                return "application/json; charset=utf-8";
+            }
+
+            @Override
+            public byte[] getBody() throws AuthFailureError {
+                try {
+                    return requestBody == null ? null : requestBody.getBytes("utf-8");
+                } catch (UnsupportedEncodingException e) {
+                    VolleyLog.wtf("Unsupported Encoding while trying to get the bytes of %s using %s", requestBody, "utf-8");
+                    e.printStackTrace();
+                    return null;
+                }
+            }
+
+            @Override
+            protected Response<String> parseNetworkResponse(NetworkResponse response) {
+                String responseString = "";
+                if (response != null) {
+                    responseString = String.valueOf(response.statusCode);
+                    // can get more details such as response.headers
+                }
+                return Response.success(responseString, HttpHeaderParser.parseCacheHeaders(response));
+            }
```

Comment:
Is it really necessary with all of these overridden methods?

Reply:
jupp. siden vi ikke sender noe tilbake så måtte jeg hacke det litt til for ikke bare få feilmelding. Det er ikke pent men det funker